ggml/src/ggml-cuda/common.cuh

```diff
@@ -864,6 +844,7 @@ struct ggml_cuda_graph {
 struct ggml_backend_cuda_context {
     int device;
     std::string name;
+    int sm_count;
```

Comment:
Instead of adding a property with duplicated information, please retrieve it as is done e.g. [here](https://github.com/ggml-org/llama.cpp/blob/1d72c841888b9450916bdd5a9b3274da380f5b36/ggml/src/ggml-cuda/mmq.cuh#L3544-L3547).

Reply:
> Instead of adding a property with duplicated information, please retrieve it as is done e.g. [here](https://github.com/ggml-org/llama.cpp/blob/1d72c841888b9450916bdd5a9b3274da380f5b36/ggml/src/ggml-cuda/mmq.cuh#L3544-L3547).

Thanks, was unaware of this design pattern of ggml.